smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptSettings.java

```diff
@@ -110,6 +110,20 @@ public static TypeScriptSettings from(Model model, ObjectNode config, ArtifactTy
         return settings;
     }
 
+    private String getDefaultDescription() {
+        String defaultDescription = getPackageName();
```

Comment:
This is no longer coming from `settings` or further up to `config`, are we sure it's correct? I don't see a test change to know for sure.

Reply:
line 93 sets the packageName on `settings` from `config`, and getDefaultDescription instance method is actually picking up that value, so @kstich I think it is what you expected. But to Kevin's point, you can add tests that assert the behavior for client and server cases.